PatternPal/PatternPal.sln

```diff
@@ -8,6 +8,9 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "10 - Back-end", "10 - Back-end", "{25C2E758-AC65-4A3A-A11A-2D3510C01BB8}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "20 - Front-end", "20 - Front-end", "{D8082781-8981-4B17-9E4A-5C25EF7E3892}"
+	ProjectSection(SolutionItems) = preProject
```

Comment:
Again; this should be removed. This is wat causes `IResult.cs` to show op separately in the solution explorer.